lib/ansible/module_utils/powershell/Ansible.ModuleUtils.Backup.psm1

```diff
@@ -18,7 +18,30 @@ Function Backup-File {
     Process {
         $backup_path = $null
         if (Test-Path -LiteralPath $path -PathType Leaf) {
-            $backup_path = "$path.$pid." + [DateTime]::Now.ToString("yyyyMMdd-HHmmss") + ".bak"
```

Comment:
I think until we are ready to try and integrate the backup file template from the config we should just leave it how it is today. Just need to make sure we document in the config entry that the config option does not apply to Windows.

Reply:
this is my attempt to integrate it, i'm just missing the piece in how to pass it to the function